corehq/apps/locations/tests/test_locations.py

```diff
@@ -384,3 +388,40 @@ def test_custom_data(self):
             },
             self.loc.sql_location.reporting_group_object().metadata
         )
+
+    def test_location_fixture_generator(self):
```

Comment:
might be worthwhile to put a comment in here since it's testing a very specific regression